### PR TITLE
Update installation instructions

### LINKED ISSUE #157: Add documentation about installing and using the GUI
Now that the lightweight GUI has been added (see #152 ), documentation needs to be added explaining:

- [ ] How to install/download the GUI version of MUSE
- [ ] How to create a standalone MUSE version in the developers instructions
- [ ] How to run the GUI version of MUSE

### LINKED ISSUE #158: Be explicit about activating the environment when developing MUSE
It seems it is not completely clear that, when developing MUSE, you should create and activate a virtual environment - [despite saying so in the instructions](https://muse-os.readthedocs.io/en/flow_charts/installation.html#instructions-for-development). This should be emphasized to avoid any ambiguity.

### LINKED ISSUE #159: Add documentation about how to seek support and submit bug reports
There are a couple of ways to seek support when using MUSE:

- The GitHub issues
- The mailing list

Documentation should include a clear section on when and how to use these two resources, and clear links should be available in a prominent location of the documentation.

### PR DESCRIPTION
**Pull request by dalonsoa**
# Description

This PR reorganises the installation instructions to make clear the different options and help the user to choose the most appropriate one. It also includes a section on installing the standalone version of MUSE - which is not available yet, but it will when we release v1.1.0 after merging this documentation - as well as a section on "Getting help". 

The built documentation can be found in https://muse-os.readthedocs.io/en/update_instalation_instructions/

Close #157 
Close #158 
Close #159 

Supersedes #167 

## Type of change

Please add a line in the relevant section of
[CHANGELOG.md](https://github.com/SGIModel/StarMuse/blob/development/CHANGELOG.md) to
document the change (include PR #) - note reverse order of PR #s.

- [x] Update documentation (non-breaking change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Optimization (non-breaking, back-end change that speeds up the code)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (whatever its nature)

## Key checklist

- [x] All tests pass: `$ python -m pytest`
- [x] The documentation builds and looks OK: `$ python -m sphinx -b html docs docs/build`

## Further checks

N/A

- [ ] Code is commented, particularly in hard-to-understand areas
- [ ] Tests added that prove fix is effective or that feature works

### CODE REVIEW

File: docs/installation/normal-installation.rst [hunk 1]
```diff
@@ -232,189 +228,27 @@ This will run a default, example model, completing after reaching year 2050. The
...
-Alternative installation instructions
--------------------------------------
-
-If you don't want to use ``pyenv`` or ``pipx``, or if you are having trouble with those tools, there are a couple of alternatives.
-
-Installing Anaconda Python
-~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-We have chosen ``pyenv`` above because it is extremely lightweight and unobtrusive with your operative system. However, you might want to consider a more fully fledged Python distribution like Anaconda, specially if your work involved non-python packages or a lot of data science and machine learning tools.
-
```
Comment (sgiarols) [inline docs/installation/normal-installation.rst:270]: Maybe this paragraph on pyenv could be joined with lines 264. Then, there is a clear separation with the section on Anaconda

**Review by sgiarols — APPROVED** [review]
It looks good, only have some notes on the names used for indicating the installation types

File: docs/installation/standalone-muse.rst [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+To get and install the standalone version of MUSE simply:
+
+- Go to the `releases page in GitHub <https://github.com/SGIModel/MUSE_OS/releases>`_ and download the standalone bundle for the release you are interested. Note that **there are bundles only from MUSE v1.1.0 onwards**, and it might be that not all releases have a bundle.
+- Unzip it to a known location in your drive, eg. ``Documents/Programs/MUSE``
+
+Inside the resulting directory you will see many files and documents related to Python and MUSE dependencies. Look for ``muse.exe`` and ``muse_gui.exe``, which are the two executables of interest, one to run MUSE in the terminal (and therefore only makes sense to execute from the terminal!) and another to run the graphical user interface (GUI) of MUSE. Just double click in ``muse_gui.exe`` and you should see the main MUSE GUI, like the following:
```
[the remark this answers is not in the document]
Comment (dalonsoa) [reply docs/installation/standalone-muse.rst:13]: Where the executables are will depend on where the user unzip the downloaded file, so we cannot indicate the full path in the documentation.

File: docs/installation/standalone-muse.rst [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+    .. image:: ../figures/muse_gui.png
+       :width: 500
+       :align: center
+       :alt: Window of the GUI version of MUSE
+
+
+Making MUSE more accessible
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
[the remark this answers is not in the document]
Comment (dalonsoa) [reply docs/installation/standalone-muse.rst:22]: Done!

File: docs/installation/index.rst [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+The :ref:`alternative-installation` gives your more flexibility in using and expanding MUSE, enable its use programmatically, i.e. within a python script or Jupyter notebook, as well as creating plugins with arbitrary complexity and dependencies. This comes at the cost of a more complex installation and running process that requires creating and activating a virtual environment manually.
+
+The :ref:`developers` method should only by used by developers, i.e. those who want to actively contribute to the MUSE code base. It involves the installation of other tools, like git or pandoc, as well as the need to follow some instructions on how the new code is formatted, the documentation created, etc.
+
+.. toctree::
+   :maxdepth: 1
+
+   standalone-muse
+   normal-installation
+   alternative-installation
```
[the remark this answers is not in the document]
Comment (dalonsoa) [reply docs/installation/index.rst:37]: Done!

File: docs/installation/normal-installation.rst (hunk 1 above)
Comment (dalonsoa) [reply docs/installation/normal-installation.rst:270]: Done!

**Pull request merged**